### DISCUSSION TITLE
Performance of this package

### DISCUSSION THREAD

**Discussion by farshadff** [spatie/laravel-pdf#73] [Q&A] [answered]
i am using laravel `dompdf` right now and with that package i can generate like 400 pages pdf at once . if  i want to go higher it throws me errors which i read its related to its performance . what i wanted to ask is that how is the performance of this package comparing to others ?? can we count it slower or faster . i know this question may be a bit generic but i need to know should i invest on this package or not . thanks in advance for your help

**Comment by freekmurze** [comment, marked as answer]
See https://spatie.be/docs/laravel-pdf/v1/alternatives

**Comment by farshadff** [comment]
thanks sir i think i have my answer

**Reply by mailnike** [reply]
Hi! Which package is best for generating large volumes of PDFs?
